slick-testkit/src/test/scala/slick/test/compile/NestedShapesTest.scala

```diff
@@ -17,14 +17,14 @@ class NestedShapesTest {
     implicitly[Shape[NestedShapeLevel, (Rep[Int], (Int, Rep[String])), _, _]]
 
     // Flat and Nested alike, fully specified
-    implicitly[Shape[FlatShapeLevel, Int, Int, ConstColumn[Int]]]
```

Comment:
Have these test cases been changed because otherwise they did not compile?
If so, then this wil probably break some use cases.

Would it be possible to restore this test?

Reply:
Sorry, these test cases cannot be recovered at this stage, I will try to describe their impact.